src/main/kotlin/net/axay/hglaborlobby/functionality/StatsDisplay.kt

```diff
@@ -0,0 +1,102 @@
+package net.axay.hglaborlobby.functionality
+
+import de.hglabor.utils.noriskutils.HologramUtils
+import kotlinx.coroutines.launch
+import net.axay.hglaborlobby.data.database.HGStats
+import net.axay.hglaborlobby.data.database.PlayerNameCache
+import net.axay.hglaborlobby.database.DatabaseManager
+import net.axay.hglaborlobby.database.mongoScope
+import net.axay.kspigot.chat.KColors
+import net.axay.kspigot.extensions.bukkit.info
+import net.axay.kspigot.extensions.console
+import net.axay.kspigot.items.itemStack
+import net.axay.kspigot.items.meta
+import net.axay.kspigot.runnables.sync
+import net.axay.kspigot.runnables.task
+import net.md_5.bungee.api.ChatColor
+import org.bukkit.Bukkit
+import org.bukkit.Location
+import org.bukkit.Material
+import org.bukkit.entity.ArmorStand
+import org.bukkit.inventory.meta.SkullMeta
+import org.litote.kmongo.*
+
+object StatsDisplay {
```

Comment:
ist das mit den zu vielen armor stands jetzt gefixt?

Reply:
ja ich hatte das nicht in die hologram liste gepackt